src/languages/es.ts

```diff
@@ -3717,6 +3717,13 @@ export default {
     },
     subscription: {
         mobileReducedFunctionalityMessage: 'No puedes hacer cambios en tu suscripción en la aplicación móvil.',
+        billingBanner: {
+            preTrial: {
+                title: 'Iniciar una prueba gratuita',
+                subtitle: 'Para empezar, ',
+                subtitleLink: 'completa la lista de configuración aquí',
```

Comment:
Did we get this translation verified?

Reply:
Yes, it was verified [here](https://docs.google.com/document/d/1skGyubv96LtirPTiCQC41jT6b65UCh0M6y_J3acqVAY/edit#heading=h.lldyzai9bgtt).